### PR TITLE
Adding reporting-related documentation for Test Plans

### PR DESCRIPTION
**Pull request by shaaradmsft**
Added documentation for five of Test Plans related reporting scenarios that cover the progress of one or more test plans.

TODO: Update the monikerRange in metadata of documentation files.

### CODE REVIEW

File: docs/report/powerbi/sample-testplans-aggregatedatlevel.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+description: Sample Power BI queries to generate aggregated view for test suites
+ms.prod: devops
+ms.technology: devops-analytics
+ms.reviewer: ravishan
+ms.manager: pkuma
+ms.author: shdalv
+ms.custom: powerbisample
+author: KathrynEE
+ms.topic: sample
+monikerRange: '>= azure-devops-2019' --TODO
```
Comment (shaaradmsft) [inline docs/report/powerbi/sample-testplans-aggregatedatlevel.md]: I request guidance for filling out this field, could you please provide more information about what monikerRange is?

**Comment by Jak-MS** [comment]
@shaaradmsft - if you are microsoft employee and content professional, can you associate your github and microsoft accounts? 

@KathrynEE - can you take a look or let us know if you're not the correct reviewer for this?

Additional note: this PR is too large for the public repo, and should be submitted via the private repo so we can take advantage of the validation, proofing, and staging tools there. if the files are approved, we'll need to move the commits in this PR.

Thanks.

**Comment by shaaradmsft** [comment]
Hi @Jak-MS, yes, I am Microsoft employee. This account is associated with my Microsoft email address and I have made the email visible. (I hope that's what you meant by associating GitHub and Microsoft accounts).

Also, since this is my first PR in this repo, could you please point me to the private repo that is used?

**Comment by Jak-MS** [comment]
hi @shaaradmsft - most private repos have the same name as the public ones with `-pr` at the end, but in this case the private repo is [azure-devops-docs-pr](https://github.com/MicrosoftDocs/azure-devops-docs-pr)

here's some information about using the private repos:

> Microsoft authors and contributors need to make content contributions through the private repository so the content can be staged and validated by the current validation rules.  
> 
> After you review staging and fix all validation issues, you can tag an author for review or add the sign-off comment to let PR reviewers know that the updates are ready to be merged.  
> 
> Read about [authors working in the public repo](https://review.docs.microsoft.com/en-us/help/contribute/contribute-authors-in-public-repo?branch=master) for more information.  
> 
> If you need help getting set up to work in the private repo, see [How to get to the private repo](https://review.docs.microsoft.com/en-us/help/contribute/contribute-authors-in-public-repo?branch=master#how-to-get-to-the-private-repo). You can also take a look at the [GitHub account setup page](https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-setup-github?branch=master) for additional details on making sure you're set up.

**Comment by Jak-MS** [comment]
@shaaradmsft - if you've opened a PR for this in the private repo, can you link to it here?  and can we close this PR? thanks.

**Comment by shaaradmsft** [comment]
Yes @Jak-MS , the private repo PR is here - https://github.com/MicrosoftDocs/azure-devops-docs-pr/pull/542
Thanks!

**Pull request closed without merging**